public/locales/en/translation.json

```diff
@@ -228,17 +228,50 @@
     "title":"Retroactive Unemployment Certification",
     "p1-multiple": "You are on Week {{weekForUser}} of {{numberOfWeeks}} to certify.",
     "form-header": "Week {{weekForUser}}: {{weekString}}",
-    "q-tooSick": "Were you too sick or injured to work?",
-    "qhelp-tooSick": "You must be well enough to work every day of the week to receive full benefits.",
-    "q-tooSickNumberOfDays": "Enter the number of days (1 through 7) you were unable to work.",
-    "q-fullTime": "Was there any reason (other than sickness or injury) that you could not have accepted full-time work each workday?",
-    "qhelp-fullTime": "Available means you are ready and willing to accept work that matches your occupational skills and educational background.",
-    "q-didYouLook": "Did you look for work?",
-    "qhelp-didYouLook": "You must look for work, according to your Notice of Unemployment Insurance Award. Work searches may include in-person, mail, telephone, or Internet contacts with employers.",
-    "q-refuseWork": "Did you refuse any work?",
-    "qhelp-refuseWork": "You will not be penalized for however you answer this question, due to COVID-19.",
-    "q-schoolOrTraining": "Did you begin attending any kind of school or training?",
-    "qhelp-schoolOrTraining": "Answer ‘yes’ only if school or training began during this week.",
+    "questions": {
+      "ui-full-time": {
+        "tooSick": "Were you too sick or injured to work?",
+        "help-tooSick": "You must be well enough to work everyday of the week to receive full benefits. Check ‘yes’ if a workday was missed due to being sick or injured.",
+        "tooSickNumberOfDays": "If Yes, enter the number of days (1 - 7) you were unable to work.",
+        "help-tooSickNumberOfDays": "You must report the numbers of days that you could not work during this week. Unemployment benefits are paid according to the number of days you are able to work this week.",
+        "couldNotAcceptWork": "Was there any reason (other than sickness or injury) that you could not have accepted full-time work <strong>each workday</strong>?",
+        "help-couldNotAcceptWork": "You must be available for work to receive unemployment benefits. Available means you are ready and willing to accept work that matches your occupational skills and educational background. If part-time work was refused for any reason other than being sick or injured check ‘yes’.",
+        "didYouLook": "Did you look for work?",
+        "help-didYouLook": "The work search requirements on your Notice of Unemployment Insurance Award must be followed. Work searches may be in-person, mail, telephone, or Internet contacts with employers.",
+        "refuseWork": "Did you refuse any work?",
+        "help-refuseWork": "If work was refused, a determination will be scheduled. Union members answer ‘yes’ if a union referral to a job was refused.",
+        "schoolOrTraining": "Did you <strong>begin</strong> attending any kind of school or training?",
```

Comment:
The <strong> is not being rendered right now. I removed it rather than fixing it since we're still waiting on final text.

Reply:
Oh, I think it is? I updated it to use the Trans component.